declarations_site/spotter/utils.py

```diff
@@ -234,8 +239,15 @@ def send_found_notify(notify):
         'query': notify.task.query,
         'decl_list': load_declarations(notify.new_ids),
         'found_new': notify.found_new,
-        'site_url': settings.EMAIL_SITE_URL,
+        'site_url': settings.SITE_URL,
     }
+    if notify.email and notify.email.endswith('.chatbot'):
```

Comment:
chatbot is not obvious domain? create separate field for channel is not about alter notify table, its about change base user model.